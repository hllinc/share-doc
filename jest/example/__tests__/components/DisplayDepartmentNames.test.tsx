import * as React from 'react'
import { mount } from 'enzyme'

import { DisplayDepartmentNames, IProps } from '../../src/components/DisplayDepartmentNames'

describe('component/DisplayDepartmentNames', () => {
  const department: IProps = {
    level1Name: 'aaa',
    level2Name: 'bbb',
    level3Name: 'ccc',
  }
  const wrap = () => mount(<DisplayDepartmentNames {...department} />)

  it('显示多级部门名称', () => {
    const app = wrap()
    expect(app.html()).toMatchSnapshot()
    app.unmount()
  })
})
