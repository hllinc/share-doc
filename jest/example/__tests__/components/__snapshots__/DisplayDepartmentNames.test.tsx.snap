// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/DisplayDepartmentNames 显示多级部门名称 1`] = `"<div>aaa<br>bbb<br>ccc</div>"`;
